diff --git a/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js b/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js
index e0a8082..7cb9f1e 100644
--- a/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js
+++ b/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js
@@ -353,7 +353,7 @@ class NsisTarget extends core_1.Target {
         await this.executeMakensis(defines, commands, sharedHeader + (await this.computeFinalScript(script, false, archs)));
         // http://forums.winamp.com/showthread.php?p=3078545
         // TODO: remove workaround when wine is fully upgraded to 11
-        if ((0, macosVersion_1.isMacOsCatalina)()) {
+        if (isWin || (0, macosVersion_1.isMacOsCatalina)()) {
             try {
                 await nsisUtil_1.UninstallerReader.exec(installerPath, uninstallerPath);
             }
